docker/README.md

```diff
@@ -0,0 +1,66 @@
+# armadietto [![Build Status](https://secure.travis-ci.org/remotestorage/armadietto.svg)](http://travis-ci.org/remotestorage/armadietto) [![js-semistandard-style](https://img.shields.io/badge/code%20style-semistandard-brightgreen.svg?style=flat-square)](https://github.com/Flet/semistandard) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/0eaafdf96ebb47a9ac462bcf6a7ccb06)](https://www.codacy.com/app/lesion/armadietto?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=remotestorage/armadietto/&amp;utm_campaign=Badge_Grade)
+
+> ### :warning: WARNING
+> Please do not consider `armadietto` production ready, this project is still
```

Comment:
Not an issue with this PR, but wondering out loud, what is the gate at which we consider Armadietto production ready?  I think hard to assess how many production workloads are already running on top of it?

Reply:
Good question! I think it merits its own issue or forums topic.